tsconfig.json

```diff
@@ -14,16 +14,21 @@
     "jsx": "preserve",
     "incremental": true,
     "baseUrl": ".",
+    "paths": {
```

Comment:
Proč je tohle lepší než současné řešení? Už teď můžu importovat relativně od kořene repa, například `src/foo`.

Reply:
Ta path jde pripadne zmenit. Ja jsem to puvodne chtel nastavit na `/src`, ale ty mas /components mimo. Takhle se to tvari, jako ze tu `components` mas exportovanou jako npm balicky. Coz je beznej zpusob pouzivani monorepa. (ale nepral bych ti to, neda se v tom proklikavat)

Kdyby si chtel dat components pod /src, tak by bylo idealni mit nastaveni te path ~/ na /src.